beta-src/src/components/ui/WDGameVoteButtons.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { useState } from "react";
+import Grid from "@mui/material/Grid";
+import GameVote from "../../enums/GameVote";
+import WDButton from "./WDButton";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+
+const WDGameVoteButtons = function (): React.ReactElement {
+  const [drawSelected, setDrawSelected] = useState(false);
+  const [pauseSelected, setPauseSelected] = useState(false);
+  const [cancelSelected, setCancelSelected] = useState(false);
```

Comment:
i still feel like this state is going to be raised to the parent, because i think when they make a vote it reflects in the table below these buttons.